README.md

```diff
@@ -1,19 +1,27 @@
 # shifthelper_webinterface
 The webinterface for the shifthelper
 
+## Technology stack
 
-# how to start:
+We are using
 
- - clone
- - cd into 
+* twitter bootstrap for responsivness (make it look good on mobile)
+* react (without jsx) for the ui component
+* socketio for the websocket (new alerts pop up without refreshing the page)
 
-# 
-Build the container:
+React using pure javascript avoids a more complicated compilation step but 
+results in more boilerplate code.
```

Comment:
This seems to be the answer to some question like: "Why is there so much boilerplate?" ... I am not advanced enough to have even gotten to this question. Can you elaborate a bit, where this boilerplate code is?